### DISCUSSION TITLE
What is Inst. Trim on RadioMaster MT12

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#4925] [Q&A] [answered]
As stated in the title, there is an action called Inst. Trim under Special Functions. What does it do?

**Comment by pfeerick** [comment, marked as answer]
> Inst. Trim (Instant Trim)- Sets all trims to the current values of their
respective sticks.

https://manual.edgetx.org/bw-radios/model-select/special-functions
